src/StdinFileInfo.php

```diff
@@ -46,6 +46,8 @@ public function getATime()
 
     public function getBasename($suffix = null)
     {
+        unset($suffix);
```

Comment:
PHPMD complaining about unused parameter without that. And actually thanks to that rule we removed few parameters that are not in use indeed. Trade off.